test/runTests.cpp

```diff
@@ -586,11 +615,39 @@ bool runTests() {
     return allTestsPassed;
 }
 
+
+/**
+ * Tests if AlphabetTooBig exception is thrown when the alphabet size is too big
+ */
+bool testException(){
```

Comment:
As I mentioned above, could we just make this to be one of the "specific" tests?